src/plugin/Manager.cc

```diff
@@ -94,10 +94,22 @@ void Manager::SearchDynamicPlugins(const std::string& dir)
 
 	// Check if it's a plugin directory.
 
-	const std::string magic = dir + "/__bro_plugin__";
+	bool bro_plugin = false;
+	std::string magic = dir + "/__zeek_plugin__";
+
+	if ( ! util::is_file(magic) )
+		{
+		bro_plugin = true;
+		magic = dir + "/__bro_plugin__";
+		}
 
 	if ( util::is_file(magic) )
 		{
+		if ( bro_plugin )
+			reporter->Warning(
+				"__bro_plugin__ is deprecated and will be unsupported in Zeek 7.1. Check to ensure "
+				"that the version of zkg in use is newer than x.y.z.");
+
```

Comment:
Even if I was the one that started the [discussion](https://github.com/zeek/cmake/pull/86/files#r1247997933): I'm now thinking it's not worth the hassle. The CMake part that generates `__bro_plugin__` or `__zeek_plugin__` is part of a Zeek installation tree so in Zeek it should be clear which one to expect.

The two scenarios that comes up:
* an older `zkg` version will not recognizing `__zeek_plugin__`: It cannot disable plugins. `zkg install` of a plugin should continue to work as expected as the initial `__zeek_plugin__` is created through cmake, not zkg. Unclear how many users actually use `zkg unload` for plugins and also, how many of those use a distro provided zkg. This is where I think "unlikely to hit anyone".

* A newer `zkg` not recognizing `__bro_plugin__` anymore. This seems like we we can/should add backwards compat on the zkg side for that, so that it moves either `__bro_plugin__` or `__zeek_plugin__` depending on which exists.

Reply:
I brought it up in Slack here: https://zeekorg.slack.com/archives/CSZBXF6TH/p1689193958220359

We'll see if there are any replies.